### PR TITLE
Add VideoPress block test cases

### PR DESCRIPTION
**Pull request by fluiddot**
This PR adds six test cases to cover the VideoPress block functionality that isn't covered with automated tests.

### CODE REVIEW

**Review by SiobhyB — APPROVED** [review]
LGTM, thank you for covering the main cases we need to have manually checked @fluiddot 🙇‍♀️

**Comment by fluiddot** [comment]
@SiobhyB I noticed when checking the test suites that the VideoPress test case links are wrong. Seems I wrongly added an extra `.md` when creating them 🤦, sorry for that. I'll fix this in https://github.com/wordpress-mobile/test-cases/pull/138.

File: test-cases/gutenberg/videopress.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+
+# VideoPress Block - Test Cases
```
Comment (dcalhoun) [inline test-cases/gutenberg/videopress.md:2]: I noticed that the Story block resides in `test-cases/jetpack`. Would it make sense to relocate VideoPress to that location or relocate Story to `test-cases/gutenberg`? The current structure is a bit confusing.

Comment (fluiddot) [reply test-cases/gutenberg/videopress.md:2]: Ah, good catch. Definitely, the VideoPress block test cases should be under `jetpack` folder not `gutenberg`. I'll open a PR to correct this. Thanks for the call out 🙇 !

Comment (fluiddot) [reply test-cases/gutenberg/videopress.md:2]: [Here is the PR with the fix.](https://github.com/wordpress-mobile/test-cases/pull/140)

**Pull request merged**